### PR TITLE
feat(Navbar): logo in Navbar

### PR DESCRIPTION
**Pull request by shiburagi**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -15,3 +15,5 @@ selenium-debug.log
 *.ntvs*
 *.njsproj
 *.sln
+
+package-lock\.json
```
Comment (nazreen) [inline .gitignore:19]: thanks for this. we already have yarn.lock and there should only be one.

**Comment by nazreen** [comment]
@shiburagi this issue is about adding logo to the Navbar. sorry if I didn't elaborate it enough.

can you explain why the change for the position of the collapse arrow?

I will attach here the link to the logo icon that should be in the Navbar, and also an example.

Below is example of logo in Navbar for Desktop
screen shot 2018-07-19 at 6 07 38 pm

Below is example for mobile:
screen shot 2018-07-19 at 6 08 14 pm

**Comment by shiburagi** [comment]
got it!

**Comment by shiburagi** [comment]
can u attach the logo file/link here??

**Comment by nazreen** [comment]
openpromises logo grey_main logo

@shiburagi  sorry for the lack of info. for any other issues feel free to ping me first for details that you feel are missing.

**Comment by nazreen** [comment]
@shiburagi  can you revert the change you made to the collapse icon ? thanks!

**Pull request closed without merging**